### DISCUSSION TITLE
Multi Choice Type

### DISCUSSION THREAD

**Discussion by Khanshab95** [community/community#87703] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

This is one of the most used script types in the chatbot. Whenever you want to create a menu or a set of options for the user to choose, then use the Multi Choice type. You can add an option by pressing the + button and remove the same using the - button.

**Comment by ghostinhershell** [comment]
Hi @Khanshab95, we're happy you're here! You are more likely to get a useful response if you are explicit about what your question, feedback request or project entails. Giving a few more details might help someone give you a nudge in the right direction.

**Comment by Khanshab95** [comment]
Ok but
